libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -81,14 +81,14 @@ typedef struct MQTTPacketInfo      MQTTPacketInfo_t;
  * data off the network.
  *
  * @param[in] context The network context provided with this function.
- * @param[out] pBuffer Buffer to receive network data.
- * @param[in] bytesToRecv Bytes to receive from the network. pBuffer must be at
+ * @param[out] pFixedBuffer Buffer to receive network data.
+ * @param[in] bytesToRecv Bytes to receive from the network. pFixedBuffer must be at
  * least this size.
  *
  * @return The number of bytes received; negative value on failure.
  */
 typedef int32_t (* MQTTTransportRecvFunc_t )( NetworkContext_t context,
-                                              void * pBuffer,
+                                              void * pFixedBuffer,
```

Comment:
I probably wouldn't change this, since this typedef will be removed from this file by #1036 anyway. Also, doesn't the transport receive take in a buffer of bytes anyway, not a pointer to a MQTTFixedBuffer_t?

Reply:
oops I wasnt supposed to change that one thanks